test/vagrant-service-manager/installer_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+require 'vagrant/util/downloader'
+
+module VagrantPlugins
```

Comment:
Awesome, with the _Vagrantfile_ removed the tests work. I get test failures, but that's due to OS specific assertions. 

Like this a lot.